_about/PULL_REQUESTS.md

```diff
@@ -16,7 +16,9 @@ expect:
    modify the code, and [create a pull request](https://help.github.com/en/articles/creating-a-pull-request-from-a-fork).
    You are welcome to submit your pull request for commentary or review before
    it is fully completed by creating a [draft pull request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests).
-   Please include specific questions or items you'd like feedback on.
+   Please include specific questions or items you'd like feedback on. 
+
+1. Our documentation utilizes the `tfplugindocs` tool. When making updates to documentation, please only edit the `.md.tmpl` files and execute the `tfplugindocs generate` command to ensure the changes are reflected in the `.md` files. 
```

Comment:
Minor recommendation. Since we mention `tfplugindocs` in the documentation, it might make sense to make it a link, so a reader can simply click on it instead of using their favorite search engine. :) We always assume that they read it the first time and are not aware of all the tools we use.

```
[`tfplugindocs`](https://github.com/hashicorp/terraform-plugin-docs)
```

Reply:
Agreed, thanks!